src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -84,6 +84,9 @@ public CaseDTO createNewCase(CaseRequestDTO request) throws CTPException {
       log.with("caseId", newCase.getId())
           .with("primaryCaseType", caseType)
           .debug("Created new case");
+      String addressLevel =
```

Comment:
The setting of the addressLevel can be combined into a one liner with a ternary, and should be moved into the createCase method

Reply:
done